wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -1130,11 +1130,11 @@ private void runS3cmd(String src, String dest) throws ModelException {
     }
 
     private void downloadDraftModel() throws ModelException, IOException {
-        String draftModelId = prop.getProperty("option.speculative_draft_model");
```

Comment:
We should not rename this, since it breaks the current experience os SD

Reply:
Sorry, I meant to raise this in draft. Current speculative decoding experience through the handler won't download an s3 model. I brought this up for discussion, we can either rename if this hasn't been used yet (https://github.com/deepjavalibrary/djl-serving/pull/1693), or we add functionality that follows the properties naming for `option.draft_model_id`.

Reply:
I found it in `lmi_dist`, missed it the first time, so I am going to close this and rethink how to unify the CX for these properties and make sure they function the same.